### PR TITLE
Added Svelte Vite Tailwind Template

### PR DESCRIPTION
**Pull request by Lukem121**

### CODE REVIEW

**Comment by swyxio** [comment]
Deploy Preview is done!

Built with commit 135eb91368732d7322cff3514860c9ea5353d06e

https://deploy-preview-424--svelte-community.netlify.app

File: data/code.yml [hunk 1]
```diff
@@ -8,6 +8,13 @@ resources:
     stars: 279
     last_updated: '2021-03-12T07:43:35Z'
     downloads: 2584
+	- name: '`svelte-vite-tailwind-template`'
```
Comment (swyxio) [inline data/code.yml]: image

spaces not tabs

**Review by swyxio — CHANGES_REQUESTED** [review]
spaces

**Comment by Lukem121** [comment]
I have removed the tabs and replaced with spaces, thanks.

**Review by swyxio — APPROVED** [review]

**Pull request merged**